src/constants.js

```diff
@@ -5,3 +5,4 @@ export const ARTICLES_PAGE_URL = "/ArticlesPage";
 export const HOME_PAGE_URL = "/HomePage";
 export const ADD_ARTICLE_URL = "/AddArticle";
 export const ARTICLE_DETAILS_URL = "/ArticleDetails";
```

Comment:
please, try to follow REST naming convention.
for example:
- `AddArticle` would be `articles/new`
- `ArticleDetails` would be `articles/{id}`
and so on.. 

have a quick view of this post https://restfulapi.net/resource-naming/

Reply:
Done